docs/propertyestimator.rst

```diff
@@ -20,15 +20,15 @@ processes on a cluster:
 
 .. code-block:: python
 
-    estimator = PropertyEstimator(nworkers=10) # NOTE: multiple backends will be supported in the future
-    computed_properties = estimator.computeProperties(dataset, parameter_sets)
+    estimator = PropertyEstimatorClient(nworkers=10) # NOTE: multiple backends will be supported in the future
+    computed_properties = estimator.computeProperties(dataset, force_fields)
```

Comment:
Aren't we sticking to [PEP8](https://www.python.org/dev/peps/pep-0008/) convention here? That would be `estimator.compute_properties()` instead of `.computeProperties()`.

Reply:
This is perhaps best viewed on RTD: https://property-estimator.readthedocs.io/en/latest/propertyestimator.html

These pages are very out of date and should be showing a warning on the page! Most of this text was just integrated from the old The-PropertyEstimator-format.md, and is not reflective of the current API. When I get to my docs push later this week / early next week all of these will be fixed up!